src/SourceTerm/Manager.cpp

```diff
@@ -288,7 +288,8 @@ auto makePointSourceCluster(ClusterMapping mapping,
                             const unsigned* meshIds,
                             seissol::initializer::LTSTree* ltsTree,
                             seissol::initializer::LTS* lts,
-                            seissol::initializer::Lut* ltsLut) -> PointSourceClusterPair {
+                            seissol::initializer::Lut* ltsLut)
+    -> seissol::kernels::PointSourceClusterPair {
```

Comment:
Is the indentation off or is this how it is intended to be?

Reply:
The formatter likes it like that. Not my decision tbh :) .